.env.example

```
@@ -0,0 +1,41 @@
+# ===============================================
+# ENVIRONMENT CONFIGURATION TEMPLATE
+# ===============================================
+
+# 1. DATABASE CONFIGURATION
+DATABASE_URL=mysql://app_user:app_password@localhost:3306/ebic-db
+REDIS_URL=redis://localhost:6379
+
+# 2. APPLICATION SETTINGS
+NEXT_PUBLIC_APP_NAME="Misurata Entrepreneurship Center"
+NEXT_PUBLIC_APP_URL=http://localhost:3000
+NEXTAUTH_URL=http://localhost:3000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXTAUTH_URL key should go before the NEXT_PUBLIC_APP_NAME key

(UnorderedKey)